packages/components/src/index.ts

```diff
@@ -27,12 +27,10 @@ export * from "./typography/OverlineText/index.ts";
 export * from "./typography/Text/index.ts";
 export * from "./utils/index.ts";
 
+export * from "@hopper-ui/styled-system";
+export { useAsyncList } from "@react-stately/data";
```

Comment:
was something conflicting here? What's the benefit of adding /data?

Reply:
we were exporting something from a package that was not in our dependencies. So i added the package to dependencies + instead of having a dependency on all of react-stately, i chose only the package i needed